spec/system/backend/solidus_stripe/orders/payments_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'solidus_stripe_spec_helper'
+
+RSpec.describe 'SolidusStripe Orders Payments', :js do
+  include SolidusStripe::BackendTestHelper
+  stub_authorization!
+
+  context 'with successful payment operations' do
+    it 'navigates to the payments page' do
+      payment = creates_authorized_payment
+      visits_payments_page
+
+      payment_displayed(payment)
+    end
```

Comment:
Love this specs, really easy to follow them 🙌 

I just have a nit to point out, i.e. the methods are using a slightly different way of stating things, deviating from the usual RSpec/Capybara style. Some examples:
- `visits_payment_page` would usually be an imperative command, like `visit_payments_page`
- `payment_displayed` would usually express an expectation, like `expects(page).to display_payment` or `expects_page_to_display_payment`

Reply:
I agree. Thinking about that, I would also make some changes to the specs related to the solidus_starter_frontend in a separate PR.

Reply:
https://github.com/solidusio/solidus_stripe/issues/248